backend/models/user.js

```diff
@@ -82,6 +82,7 @@ UserSchema.set("toJSON", {
   transform(doc, obj) {
     const ret = { ...obj };
     delete ret.password;
+    delete ret.refreshToken;
```

Comment:
What is the purpose of deleting the refresh token here?

Reply:
I realized that the route which gets all users actually returns the refresh token for every user, which is rather problematic from a security perspective. This change makes sure that the refresh token doesn't show up when a User object is converted to JSON. This change is mostly unrelated to the rest of the PR.